### DISCUSSION TITLE
Can we change the binary called when using the --search-zip flag

### DISCUSSION THREAD

**Discussion by JeffreyShran** [BurntSushi/ripgrep#1929] [Q&A] [answered]
I would like to use `rg` with `pigz` instead of `gzip` - is it possible?

For reference, this si the flag I'm looking at:
> ripgrep supports searching files compressed in a common format (brotli, bzip2, gzip, lz4, lzma, xz, or zstandard) with the -z/--search-zip flag.

**Comment by BurntSushi** [comment, marked as answer]
Not with `--search-zip`, no. You could use the `--pre` and `--pre-glob` flags though to do what you want.

See also: https://github.com/BurntSushi/ripgrep/issues/1916

**Reply by JeffreyShran** [reply]
awesome, thanks :)
